packages/picasso.js/src/web/text-manipulation/text-metrics.js

```diff
@@ -60,8 +72,8 @@ function measureTextHeight(fontSize) {
  *  fontFamily: 'Arial'
  * }); // returns { width: 20, height: 12 }
  */
-export function measureText({ text, fontSize, fontFamily }) {
-  const w = measureTextWidth(text, fontSize, fontFamily);
+export function measureText({ text, fontSize, fontFamily, fontStyle, fontWeight }) {
+  const w = measureTextWidth({ text, fontSize, fontFamily, fontStyle, fontWeight });
```

Comment:
Have you check if `fontStyle` actually makes any difference on the measured text?

Reply:
That is what I recall as well. Unexpected, but maybe we don't need it here?

Reply:
Removed!